com.unity.render-pipelines.high-definition/Editor/PostProcessing/CustomPostProcessVolumeComponentEditor.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+using UnityEngine.Rendering.HighDefinition;
+using UnityEngine.Rendering;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    /// <summary>
+    /// Base class to inherit to create custom post process volume editors.
+    /// </summary>
+    [CanEditMultipleObjects]
+    [VolumeComponentEditor(typeof(CustomPostProcessVolumeComponent))]
```

Comment:
```suggestion
    [CustomEditor(typeof(CustomPostProcessVolumeComponent))]
```

VolumeComponent Editor is deprecated, see `com.unity.render-pipelines.core\Editor\Deprecated.cs`

Reply:
Yes, I did the changes with a completely outdated version of the repo.
Thanks for the suggestion block!

Reply:
Hum, this is breaking my fix, I need to investigate why. Maybe inheritance isn't supported by default with the CustomEditor attribute :/

Reply:
Maybe, you are missing the parameter after the  type, to be true, to be used by the children?

Reply:
Ah yes, forgot about that. It works now, thanks!